src/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export type AnyObject = Record<string, any>;
+
+export type Maybe<T> = T | null;
```

Comment:
In server it represents [flow's](https://flow.org/en/docs/types/maybe/) `Maybe` type and includes `undefined`. To make it less confusing let's remove it and define explicitly in types

```suggestion
export type Maybe<T> = null | undefined | T;
```